engine/object.asm

```diff
@@ -117,6 +119,8 @@ ObjectLoop2@
             puls        a
             deca
             bne         ObjectLoop2@
+!           lda         <MemMgr_VirtualTable+VH_LVLOBJCODE1 * Map in the first page
+            sta         $FFA3
```

Comment:
Can this be surrounded by an "IFNE OBJPAGES-1" macro?

Reply:
Yes!

Reply:
Fixed